cli/README.md

```diff
@@ -13,75 +13,96 @@ mainflux-cli version
 ```
 
 ### User management
-#### Create User
+#### Create Users
 ```
 mainflux-cli users create john.doe@email.com password
 ```
 
-#### Login User
+#### Login Users
 ```
 mainflux-cli users token john.doe@email.com password
 ```
 
 ### System Provisioning
-#### Provision Device
+#### Create Devices
```

Comment:
I'd prefer using singular nouns in headings. I would assume from the title that I can pass a list of devices to this command.